Packs/LookoutCASB/Integrations/CASBIntegration/CASBIntegration.yml

```
@@ -0,0 +1,187 @@
+commonfields:
+  id: Lookout CASB
+  version: -1
+name: Lookout CASB
+display: Lookout CASB
+category: Data Enrichment & Threat Intelligence
+description: Lookout CASB Integration
+configuration:
+- display: Server URL (e.g. https://management-server.ciphercloud.com)
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Maximum number of incidents per fetch
+  name: max_fetch
```

```suggestion
  name: max_fetch
```

this variable only configured in the yml file but you dont use it at all, please limit the fetched incidents with this value

---

max_fetch is optional. There is a max limit at the server side. This field is provided for client as optional if client would like to limit the records.